apps/i18n/studio/en_us.json

```diff
@@ -52,6 +56,8 @@
   "catProcedures": "Functions",
   "catText": "Text",
   "catVariables": "Variables",
+  "caveboy": "cave boy",
+  "cavegirl": "cave girl (Jasmine)",
```

Comment:
I was blindly reproducing the way they appear in the "set sprite" block. Any idea why only some of those show the name?